hazelcast/src/test/java/com/hazelcast/executor/CompletableFutureTest.java

```diff
@@ -0,0 +1,128 @@
+package com.hazelcast.executor;
+
+import com.hazelcast.core.ExecutionCallback;
+import com.hazelcast.core.ICompletableFuture;
+import com.hazelcast.instance.HazelcastInstanceImpl;
+import com.hazelcast.instance.HazelcastInstanceProxy;
+import com.hazelcast.spi.ExecutionService;
+import com.hazelcast.spi.NodeEngine;
+import com.hazelcast.test.HazelcastTestSupport;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static org.junit.Assert.assertEquals;
+
+public class CompletableFutureTest extends HazelcastTestSupport {
```

Comment:
Can you add the following on top of your test:

```
@RunWith(HazelcastSerialClassRunner.class)
@Category(QuickTest.class)
```

This way they are going to be picked up by Maven.